model/transformer_utils.py

```diff
@@ -3,7 +3,7 @@
 
 
 def get_angles(pos, i, model_dim):
-    angle_rates = 1 / np.power(10000, (2 * i) / np.float32(model_dim))
+    angle_rates = 1 / np.power(10000, (2 * (i//2)) / np.float32(model_dim))
```

Comment:
This looks like its gonna break old model predictions?

Reply:
It's actually a revert-back